config.template/faf-clans/faf-clans.env

```diff
@@ -1,7 +1,3 @@
-API_BASE_URL=https://api.dev.faforever.com
-OAUTH_CLIENT_ID=83891c0c-feab-42e1-9ca7-515f94f808ef
-OAUTH_REDIRECT_URI=https://clans.faforever.com
-
-#VIRTUAL_HOST=clans.faforever.com
-#LETSENCRYPT_HOST=clans.faforever.com
-#LETSENCRYPT_EMAIL=admin@faforever.com
+API_BASE_URL=https://localhost:8010
```

Comment:
Does that work?

The clanapp-container can't access `https://localhost:8010`. It needs to be `https://faf-java-api:8010`
Just like another app does it... https://github.com/FAForever/faf-stack/blob/develop/config.template/faf-website/faf-website.env#L7